app/components/AccountCard/AccountCard.tsx

```diff
@@ -293,7 +294,7 @@ export default function AccountCard({
             : 0n;
     const accountBaker = accountInfo?.accountBaker;
     const stakedAmount = accountBaker ? BigInt(accountBaker.stakedAmount) : 0n;
-    const amountAtDisposal = unShielded - scheduled - stakedAmount;
+    const amountAtDisposal = unShielded - max(scheduled, stakedAmount);
```

Comment:
Make a function for this as we use it in multiple places. Then the function can also document why this is the amount that is at disposal.

Reply:
I have added a function to calculate the public amounts, as this was done multiple places. Determining the atDisposal is a part of that.